### ISSUE TITLE
配置文件是否可以提供yaml格式的？

### ISSUE THREAD

**Issue by mason0417** [hustvl/ViTGaze#10]
您好！首先感谢您开发了这么优秀的项目，它真的让我受益良多。我最近在尝试使用它，但有一个小问题想请教一下。我看到目前的配置文件是py，不知道您是否方便分享一个 YAML 格式的配置文件示例呢？

**Comment by Ambitious-idiot** [comment]
你好，运行训练脚本会在输出目录下自动生成yaml格式的配置文件`config.yaml`，你可以参考一下

**Comment by Ambitious-idiot** [comment]
这是我用configs/gazefollow.py跑出来的结果：
`dataloader:
  train: {_target_: detectron2._cfg_loaderd259.dataloader.py.__build_dataloader, batch_size: 12, distributed: true, input_size: 434, is_train: true, mask_prob: 0.5, mask_scene: true, mask_size: 31, max_scene_patches_ratio: 0.5, name: gazefollow, num_workers: 14, rand_lsj: 0.5, rand_rotate: 0.5}
  val: {_target_: detectron2._cfg_loaderd259.dataloader.py.__build_dataloader, batch_size: 64, distributed: false, input_size: 434, is_train: false, name: gazefollow, num_workers: 14}
lr_multiplier:
  _target_: detectron2.solver.WarmupParamScheduler
  scheduler: {_target_: detectron2._cfg_loaderce62.scheduler.py.get_scheduler, end_value: 0.1, start_value: 1, typ: cosine}
  warmup_factor: 0.001
  warmup_length: 0.01
model:
  _target_: modeling.meta_arch.GazeAttentionMapper
  backbone:
    _target_: modeling.backbone.build_backbone
    name: dinov2_small
    out_attn: [2, 5, 8, 11]
    return_softmax_attn: true
    use_cls_token: true
    use_mask_token: true
  classifier: {_target_: modeling.head.build_inout_head, in_channel: 384, name: SimpleMlp}
  criterion: {_target_: modeling.criterion.GazeMapperCriterion, aux_head_thres: 0.05, aux_weight: 100, use_aux_loss: true, use_focal_loss: true}
  device: cuda
  pam: {_target_: modeling.patch_attention.build_pam, embed_dim: 8, name: PatchPAM, patch_size: 14, use_aux_loss: true}
  regressor:
    _target_: modeling.head.build_heatmap_head
    deconv_cfgs:
    - {in_channels: 24, kernel_size: 3, out_channels: 16, padding: 1, stride: 1}
    - {in_channels: 16, kernel_size: 3, out_channels: 8, padding: 1, stride: 1}
    - {in_channels: 8, kernel_size: 3, out_channels: 1, padding: 1, stride: 1}
    dim: 24
    feat_type: attn
    in_channel: 24
    name: UpSampleConv
    use_conv: false
  use_aux_loss: true
optimizer:
  _target_: torch.optim.AdamW
  betas: [0.9, 0.99]
  lr: 0.0001
  params:
    _target_: detectron2.solver.get_default_optimizer_params
    base_lr: ${..lr}
    lr_factor_func: !!python/object:detectron2._cfg_loader5b89.optimizer.py.LRDecayRater {backbone_multiplier: 1.0, freeze_pe: false, lr_decay_rate: 0.65, num_layers: 12, pam_lr_decay: 1}
    overrides:
      pos_embed: {weight_decay: 0.0}
    weight_decay_norm: 0.0
  weight_decay: 0.1
train:
  amp: {enabled: false}
  checkpointer: {max_to_keep: 10, period: 2625}
  ddp: {broadcast_buffers: true, find_unused_parameters: false, fp16_compression: true}
  device: cuda
  eval_period: 5000
  init_checkpoint: pretrained/dinov2_small.pth
  log_period: 26
  max_iter: 36750
  output_dir: ./output/gazefollow
  seed: 0
`
